.github/e2e-matrix-generator.py

```diff
@@ -252,5 +252,5 @@ def build_schedule_include_cloud(engine_version_list):
         for job in include:
             job["id"] = engine + "-" + job["id"]
             print(f"Generating {engine}: {job['id']}", file=sys.stderr)
-        print(f"::set-output name={engine}Matrix::" + json.dumps({"include": include}))
-        print(f"::set-output name={engine}Enabled::" + str(len(include) > 0))
+        print(f"{engine}Matrix=" + json.dumps({"include": include}) + " >> $GITHUB_OUTPUT")
+        print(f"{engine}Enabled=" + str(len(include) > 0) + " >> $GITHUB_OUTPUT")
```

Comment:
This cannot work. It should be something like

```
        with open(os.environ["GITHUB_OUTPUT"], 'a') as f:
            f.write(f"{engine}Matrix=" + json.dumps({"include": include}) + "\n")
            f.write(f"{engine}Enabled=" + str(len(include) > 0) + "\n")
```

Reply:
Thanks Macro, I modified accordingly